cypress/page/payment-step.page.ts

```diff
@@ -19,9 +17,11 @@ class PaymentStepPage {
         cy.get(this.confirmOrderButton).click()
     }
 
-    public checkResult(): void {
-        cy.get(this.storeIsCompleted)
-          .should("have.text", this.testToCheck);
+    public getConfirmationMessage(message: string): void {
```

Comment:
In this case, as we are verifying the message, a better name would be: `verifyConfirmationMessage`